Api/Infrastructure/EndpointConfig.cs

```diff
@@ -18,10 +18,17 @@ public class EndpointConfig
             LogConfig.ConfigureLogging();
 
             var endpointConfiguration = new EndpointConfiguration(ConfigurationManager.AppSettings["Environment.ShortName"] + "_claims_invoicereceiver");
-            endpointConfiguration.EnableInstallers();
+            
+            // You should not always invoke the installers
```

Comment:
You should explain why they don't always need to run. It's pretty unclear atm.

Reply:
Installers should be run only once during deployment/setup, not executing the installers improves startup latency.

Reply:
This is a PR on his own repo so I do hope he'll read these comments :-)

Reply:
lol